src/pep257.py

```diff
@@ -1592,6 +1607,18 @@ def check_no_signature(self, function, docstring):  # def context
             if function.name + '(' in first_line.replace(' ', ''):
                 return D402()
 
+    @check_for(Function)
+    def check_capitalized(self, function, docstring):
+        """D403: First word of the first line should be properly capitalized.
+
+        The [first line of a] docstring is a phrase ending in a period.
+
+        """
+        if docstring:
+            first_word = eval(docstring).split()[0]
```

Comment:
That's a good question. I'm not 100% certain. My familiar with the tokenizer library isn't so great.